lib/board.rb

```diff
@@ -117,4 +117,19 @@ def place(ship, placement_coordinates)
     end
     #may need to add error/return value for non placement
   end
+
+  def render
+    rendered_cells = @cells.map do |coordinate, cell|
```

Comment:
Could each of these enumerable methods be a helped method outside?

Reply:
Yes! I think we definitely need to do that in the next round.